.pre-commit-config.yaml

```diff
@@ -4,6 +4,11 @@ repos:
     hooks:
     - id: black
   - repo: https://github.com/pycqa/isort
-    rev: 5.8.0  # Also stored in dev-requirements.txt; update both together!
+    rev: 5.10.1  # Also stored in dev-requirements.txt; update both together!
```

Comment:
I'm okay bumping the version of isort, but it should happen in a separate commit, and per the comment here, the version in dev-requirements.txt needs to be updated at the same time